solutions/cses-1676.mdx

```diff
@@ -56,9 +64,61 @@ int main () {
 
 </CPPSection>
 
-<JavaSection>
+<PySection>
+
+```python
+import sys
+input = sys.stdin.readline
```

Comment:
what's this for

Reply:
making input faster

Reply:
redefining input makes TC#8 0.60 seconds while without it it's 1 second, barely passing

Reply:
can you add a comment saying this is the case